### ISSUE TITLE
google form received by K9 client shows incorrect data for date fields

### ISSUE THREAD

**Issue by wombatch** [thunderbird/thunderbird-android#8042]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### App version

6.804

### Where did you get the app from?

F-Droid

### Android version

e/os 2.2 android 13 but not relevant problem exists on multiple android versions K9 6.801 and 6.804 possibly others

### Device model

fairphone 5 but not relevant problem exists on multiple hardware

### Steps to reproduce

Go to google
Create a google form with a date field
Email the form to be received by a K9 client
K9 does not correctly display the date field until reply is pressed.

### Expected behavior

Date field should be presented as such

### Actual behavior

Displays a list of all possible date combinations

### Logs

![IMG_20240731_030953_326](https://github.com/user-attachments/assets/fbc54b05-6264-401a-bed3-23aed49aebbd)

**Comment by wombatch** [comment]
Is this in the right place, no action (even acceptance) seems to have happened. When I opened it in K9 github you were very very quick to tell me that k9 is now called thunderbird and i have to open the problem here. It would be nice if it was at least acknowledged.

**Comment by wombatch** [comment]
closed in error..

**Comment by wombatch** [comment]
closed in error

**Comment by wombatch** [comment]
Still not fixed in K9 8.1

**Comment by kewisch** [comment]
It's been a while but I am acknowledging your message :) I suspect this is due to how the webview is locked down when displaying email, and maybe some HTML sanitizing. Would you happen to have an .eml file with the email you received from Google? This might make it a bit easier to reproduce.

**Comment by wombatch** [comment]
Yes, I have a copy of the email, I've edited the file a little to remove contact information but the problem seems to still be the same/similar.

**Comment by wombatch** [comment]
It's been a while I know but i kept the google forms email to check whether this was resolved. I now use /e/OS but the email client there traces its roots back to (k9) Thunderbird. I checked the latest Thunderbird and K9 (why do you still ship something called k9?) and the problem is the same is there/will there be anything done to fix it?
